src/containers/dynamicarray.d

```diff
@@ -147,29 +161,83 @@ struct DynamicArray(T, Allocator = Mallocator, bool supportGC = shouldAddGCRange
 		insert(value);
 	}
 
+	/**
+	* ~= operator overload for an array of items
+	*/
+	void opOpAssign(string op, bool checkForOverlap = true)(AppendT[] rhs)
```

Comment:
This should return a reference to itself. It is what is expected of assignment operators such as this.

```D
int a = 0;

if((a += 1) == 1)
{
}
```

Reply:
The single element `~=` does not currently return a reference to itself, so I'll change it too.